content/ko/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 컨테이너 라이프사이클 이벤트에 핸들러 연결하기
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+
+이 페이지에서는 컨테이너 라이프사이클 이벤트에 핸들러를 첨부하는 방법을 보여준다. 쿠버네티스는
+postStart 이벤트와 preStop 이벤트를 지원한다. 쿠버네티스는 컨테이너가 시작된 직후
+postStart 이벤트를 전송하고, 컨테이너가 종료되기 직전에 preStop 이벤트를 전송한다.
+컨테이너는 이벤트당 하나의 핸들러를 지정할 수 있다.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+
+<!-- steps -->
+
+## postStart 및 preStop 핸들러 정의
+
+이 연습에서는 하나의 컨테이너로 이루어진 파드를 생성한다. 컨테이너에는
+postStart와 preStop 이벤트에 대한 핸들러가 있다.
+
+다음은 파드의 구성 파일이다.
+
+{{< codenew file="pods/lifecycle-events.yaml" >}}
+
+구성 파일에 postStart 명령은 컨테이너의 `/usr/share` 디렉토리에
+`message` 파일을 쓰는 것을 확인할 수 있다. preStop 명령은 nginx
+정상적으로(gracefully) 종료한다. 이는 컨테이너가 장애로 인해 종료되는 경우에 유용하다.
```

Comment:
[원문]
> In the configuration file, you can see that the postStart command writes a `message` file to the Container's `/usr/share` directory. The preStop command shuts down nginx gracefully. This is helpful if the Container is being terminated because of a failure.

```suggestion
구성 파일에서, postStart 명령은 컨테이너의 `/usr/share` 디렉토리에
`message` 파일을 생성하는 것을 확인할 수 있다. preStop 명령은 nginx를
정상적으로(gracefully) 종료한다. 이는 컨테이너가 장애로 인해 종료되는 경우에 유용하다.
```